impexp-client-cli/src/main/java/org/citydb/cli/ImpExpCli.java

```diff
@@ -535,6 +535,9 @@ private void createPidFile() throws ImpExpException {
 
     private void loadClasses(Path path, Util.URLClassLoader classLoader) throws IOException {
         if (Files.exists(path)) {
+            if (Files.isSymbolicLink(path)) {
+                path = Files.readSymbolicLink(path);
+            }
             try (Stream<Path> stream = Files.walk(path)
```

Comment:
Why not just use `Files.walk(path, FileVisitOption.FOLLOW_LINKS)`?

Reply:
Yes, it is nicer